.circleci/config.yml

```diff
@@ -46,7 +46,8 @@ commands:
           name: Install Dependencies
           command: |
             sudo apt-get update
-            sudo apt-get install -y cmake curl
+            sudo apt-get install -y cmake curl clang llvm
+            cargo install bindgen
```

Comment:
I believe this is only required if were to use bindgen cli , we now have build-dependencies so we probably don't need this anymore.